File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    answer = []
    for arr in commands:
        i, j, num = arr[0]-1, arr[1]-1, arr[2]-1
        # index 용으로 변환
        test = sorted(array[i:j+1])
        answer.append(test[num])
        
        
    return answer

File: 프로그래머스/1/42748. K번째수/K번째수.java
import java.util.*;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        for(int y = 0; y < commands.length; y++) {
            int[] arr = commands[y];
            int i = arr[0]-1, j = arr[1]-1, k = arr[2]-1;
            int[] temp = new int[j-i+1];
            for(int x = 0; x < j-i + 1; x++) {
                temp[x] = array[i+x];
            }
            Arrays.sort(temp);
            answer[y] = temp[k];
        }

        return answer;
    }
}